lib/utils/transpile-es6.js

```diff
@@ -22,6 +22,10 @@ module.exports = function(tree, description, options) {
   }
 
   var outputTree = map(tree, '**/*.js', function(content, relativePath) {
+    if (relativePath.match(/umd\.js$/)) {
+      return '';
+    }
```

Comment:
@stefanpenner I agree, it has been a little whack-a-mole-y, but I'm glad to work on the less exciting stuff that needs to get done.

I'm in favor of removing the `.js` extension from the UMD files, but I don't have enough context to know of the side-effects.

Frankly, I still don't understand the technical differences between AMD / CJS / UMD, so I'll jump through whatever hoops I'm advised to.